modules/core/src/test/scala/com/azavea/stac4s/CatalogSpec.scala

```diff
@@ -0,0 +1,436 @@
+            title = Some("Green Band (B3)"),
+            description = Some("Green Band Top Of the Atmosphere"),
+            roles = Nil,
+            _type = Some(`image/tiff`)
+            // "eo:bands": [2],
+          ),
+          "B4" -> StacItemAsset(
+            href = "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B4.TIF",
+            title = Some("Red Band (B4)"),
+            description = Some("Red Band Top Of the Atmosphere"),
+            roles = Nil,
+            _type = Some(`image/tiff`)
+            // "eo:bands": [3],
+          ),
+          "B5" -> StacItemAsset(
+            href = "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B5.TIF",
+            title = Some("NIR Band (B5)"),
+            description = Some("NIR Band Top Of the Atmosphere"),
+            roles = Nil,
+            _type = Some(`image/tiff`)
+            // "eo:bands": [4],
+          ),
+          "B6" -> StacItemAsset(
+            href = "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B6.TIF",
+            title = Some("SWIR (B6)"),
+            description = Some("SWIR Band Top Of the Atmosphere"),
+            roles = Nil,
+            _type = Some(`image/tiff`)
+            // "eo:bands": [5],
+          ),
+          "B7" -> StacItemAsset(
+            href = "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B7.TIF",
+            title = Some("SWIR Band (B7)"),
+            description = Some("SWIR Band Top Of the Atmosphere"),
+            roles = Nil,
+            _type = Some(`image/tiff`)
+            // "eo:bands": [6],
+          ),
+          "B8" -> StacItemAsset(
+            href = "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B8.TIF",
+            title = Some("Panchromatic Band (B8)"),
+            description = Some("Panchromatic Band Top Of the Atmosphere"),
+            roles = Nil,
+            _type = Some(`image/tiff`)
+            // "eo:bands": [7],
+          ),
+          "B9" -> StacItemAsset(
+            href = "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B9.TIF",
+            title = Some("Cirrus Band (B9)"),
+            description = Some("Cirrus Band Top Of the Atmosphere"),
+            roles = Nil,
+            _type = Some(`image/tiff`)
+            // "eo:bands": [8],
+          ),
+          "B10" -> StacItemAsset(
+            href = "http://landsat-pds.s3.amazonaws.com/L8/153/025/LC81530252014153LGN00/LC81530252014153LGN00_B10.TIF",
+            title = Some("LWIR Band (B10)"),
+            description = Some("LWIR Band Top Of the Atmosphere"),
+            roles = Nil,
+            _type = Some(`image/tiff`)
+            // "eo:bands": [9],
```

Comment:
StacItemAssets should also have some `extensions` support. 
i.e. `"eo:bands": [0,1,2,3]`: https://github.com/radiantearth/stac-spec/blob/master/extensions/eo/README.md

Reply:
@jisantuc @notthatbreezy ^ `properties` would not work here as well